app/web_framework/cluster_master_application.py

```diff
@@ -33,6 +39,7 @@ def __init__(self, cluster_master):
         root_route = \
             RouteNode(r'/', _RootHandler).add_children([
                 RouteNode(r'v1', _APIVersionOneHandler).add_children([
+                    RouteNode(r'metrics', _MetricsHandler),
```

Comment:
We may want to have this endpoint disabled by default and enabled with a command line option. I'm not sure what the cleanest way of doing that is.